algorithms/linfa-clustering/src/appx_dbscan/hyperparameters.rs

```diff
@@ -99,21 +99,21 @@ impl<F: Float> AppxDbscanHyperParams<F> {
     }
 
     fn build(tolerance: F, min_points: usize, slack: F) -> Self {
-        if tolerance <= F::cast(0.) {
+        if tolerance <= F::zero() {
```

Comment:
there is also `tolerance.is_negative()`

Reply:
From the documentation it seems like `is_negative` is the `< 0` operator, not the `<= 0` operator

Reply:
yeah I should do them when I'm fully awake :flushed: